.changeset/neat-dingos-attend.md

```
@@ -0,0 +1,7 @@
+---
+"@ensnode/ensnode-sdk": patch
+"enssdk": patch
+"ensapi": patch
+---
+
+omnigraph: rename chain to chainName, remove DEFAULT chain, add Hex type to address.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)